motion/reference_model/dp_reference_model/include/dp_reference_model/reference_model.h

```diff
@@ -1,81 +1,72 @@
-/**
- * @file
- *
- * Written by Jae Hyeong Hwang
- * Modified by Øystein Solbø
- * Copyright (c) 2021 Manta AUV, Vortex NTNU.
- * All rights reserved.
- *
- *
- * Module connecting the DP-controller to the joystick_guidance and
- * the dp_guidance
- */
 #ifndef DP_REFERENCE_MODEL_H
```

Comment:
Would be nice to include a short writeup of the math here 😄

Reply:
The Reference Model definitely needs reference to relevant literature as in the [quaternion_dp_controller.h](https://github.com/vortexntnu/vortex-auv/pull/379/files#diff-6388b55262a17c9737dce46e29e17d838e4bfb3984c2a3511b9cbc6b71a32e21), if there exists something specific. Otherwise, the math should be documented in a separate channel, i.e. the Vortex Wiki.

Reply:
Set as TODO